mssw/My-List-Util/lib/Test/My/List/Util.pm

```diff
@@ -0,0 +1,227 @@
+package Test::My::List::Util;
+
+use 5.006;
+use strict;
+use warnings;
+
+use Exporter qw(import);
+use vars qw(@EXPORT $VERSION);
```

Comment:
ahh that might have been some copy paste confusion from the book clashing with module starter